### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by Owusu-Desmond**
Use HTML5 semantic tags

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        <div class="networkContent">
+          <ul class="links">
+            <li><a href="https://twitter.com/DesmondOwusuDev"><img src=" ./images/twitter.svg" alt="twitter"></a></li>
+            <li><a href="https://www.linkedin.com/in/desmond-owusu-ansah-09274a223/"><img src=" ./images/Linkedin-icon.svg" alt="Linkedin"></a></li>
+            <li><a href="https://github.com/Owusu-Desmond"><img src=" ./images/github.svg" alt="github"></a></li>
+            <li><a href="#"><img src=" ./images/M-icon.svg" alt="M-icon"></a> </li>
+            <li><a href="#"><img src=" ./images/hand-raise.svg" alt="handraise icon"></a></li>
+          </ul>
+        </div>
+      </section>
```
Comment (Kandy-Peter) [inline index.html:48]: - [ ] It is required that your template should match the one provided by Microverse. Check the [templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) and Ensure that you `fonts`, `colours` and `icons` are almost similar to the template reference.

1. **Your design**
image 
2. **Figma desing**
image

**Review by Kandy-Peter — CHANGES_REQUESTED** [review]
Hi @Owusu-Desmond,

Good job so far! I know you have worked hard on making this milestone. There is an issue that you still need to work on to go to the next project but you are almost there!

almost-there-almost


## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- You  are following GitHub actions :heavy_check_mark: 
- Good commit history :heavy_check_mark: 

## Required Changes ♻️

- [ ] As requested by the previous reviewer, Please, do ensure your PR title is descriptive and related to the milestone you've worked on also, add a summary of what you've done in the PR as what you have right now is not descriptive enough regarding the milestone you've worked on.
 - e.g Title: Portfolio Mobile Setup

```
In this project milestone, I did the following

Added the page layout in the index.html file
Added styling.
Ensured there was no linter error.
etc
```

- [ ] Please, kindly update your readme by adding the necessary header from the [microverse readme template](https://github.com/microverseinc/readme-template)

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (zieeco) [reply index.html:48]: **[REQUIRED CHANGES]**

- As required by the previous reviewer, please kindly add the missing color to the `LET'S CONNECT`
- Add the correct spacing between the social media icons.
- increase the font size of the paragraph to match the Figma design
- Remove the  second `<br>` from your h1 to make it seat properly

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Owusu-Desmond 


Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the changes, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omarsalem7 — APPROVED** [review]
Hi @Owusu-Desmond, 
# STATUS: APPROVED! 🥇 💯 
giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Implement changes correctly ✔️
- No linter errors ✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Code Review by @omarsalem7

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**